tests/test_models.py

```diff
@@ -51,3 +51,53 @@ def test_roles(self):
         person.roles.append(role)
         assert role in person.roles
         assert person.has_roles('admin')
+
+
+@pytest.mark.usefixtures('db')
+class TestCarpool:
+    """Carpool tests."""
+
+    def test_get_ride_requests_query(self, db):
+        """Test get ride requests query"""
+        carpool_1 = CarpoolFactory()
+        carpool_2 = CarpoolFactory()
+
+        person = PersonFactory()
+
+        ride_request_1 = RideRequest(
+            person = person,
+            carpool = carpool_1,
+        )
+        ride_request_2 = RideRequest(
+            person = person,
+            carpool = carpool_1,
+            status = 'approved',
+        )
+        ride_request_3 = RideRequest(
+            person = person,
+            carpool = carpool_2,
+            status = 'rejected',
+        )
+
+        db.session.add_all([
+            carpool_1,
+            carpool_2,
+            person,
+            ride_request_1,
+            ride_request_2,
+            ride_request_3,
+        ])
+        db.session.commit()
+
+        all_ride_requests = \
+            carpool_1.get_ride_requests_query().all()
+
+        assert len(all_ride_requests) == 2
```

Comment:
We can express this a little more concisely as

```
assert all_ride_requests == [ride_request_1, ride_request_2]
```

I'm also not sure we want to make assertions about order here. If we don't, we can use

```
assert set(all_ride_requests) == {ride_request_1, ride_request_2}
```

Reply:
done